src/Raven.Server/ServerWide/ServerStore.cs

```diff
@@ -1202,7 +1201,7 @@ public void IdleOperations(object state)
             {
                 try
                 {
-                    _timer.Change(_frequencyToCheckForIdleDatabases, TimeSpan.FromDays(7));
+                    _timer.Change(_frequencyToCheckForIdleDatabases, _frequencyToCheckForIdleDatabases);
```

Comment:
That was intentionally very high in the future, we don't want to have slow check that will become re-enterant.